src/jquery.multiselect.js

```diff
@@ -726,11 +763,24 @@
          return;
 
       this._positioned = false;
-      var width = this.element.outerWidth();
-      var minWidth = this._getMinWidth();
 
-      this._savedButtonWidth = width < minWidth ? minWidth : width;
-      this.$button.outerWidth(this._savedButtonWidth);
+      // this._selectWidth set in _create() for native select element before hiding it.
+      var width = this._selectWidth || this._getBCRWidth( this.element );
+      var buttonWidth = this.options.buttonWidth || '';
+      if (/\d/.test(buttonWidth)) {
+         var parsed = this._parse2px(buttonWidth, this.element);
+         var pixels = parsed.px;
+         var minimax = parsed.minimax;
+         width = minimax < 0 ? Math.max(width, pixels) : ( minimax > 0 ? Math.min(width, pixels) : pixels );
+      }
+      else // keywords
```

Comment:
Same thing with the braces on all these.

Reply:
Keywords?

Reply:
Keywords :  e.g.  'auto'    The comment is just emphasizing that keyword options drop to the else.